### PR TITLE
Change crossbow firing skills to match related firearms.

### PR DESCRIPTION
**Pull request by kevingranade**
As mentioned [a](http://smf.cataclysmdda.com/index.php?topic=2375.msg30650#msg30650) [few](http://smf.cataclysmdda.com/index.php?topic=2168.msg27703#msg27703) [times](http://smf.cataclysmdda.com/index.php?topic=5908.msg104481#msg104481) [previously](http://smf.cataclysmdda.com/index.php?topic=14564.msg300479#msg300479), the act of firing a crossbow doesn't bear any resemblance to the act of firing a bow, so using the same skill for both is nonsense.

Additionally, this provides a means to train various firearm skills without having to resort to expending valuable firearm rounds.

Thanks to @DangerNoodle for reminding me to get around to this in #21538

### CODE REVIEW

File: data/json/items/archery.json [hunk 1]
```diff
@@ -840,7 +840,7 @@
       "price" : 324000, "//" : "Modern crossbows can come in anywhere from $300 to $2K.  Large plus Antique = expensive.",
       "material" : ["iron", "wood"],
       "flags" : ["FIRE_TWOHAND", "STR_RELOAD", "PRIMITIVE_RANGED_WEAPON", "TRADER_AVOID"],
-      "skill" : "archery",
+      "skill" : "rifle",
```
Comment (kevingranade) [inline data/json/items/archery.json]: Possibly switch to launcher instead of rifle?  I'm abivalent.

Comment (Coolthulhu) [inline data/json/items/archery.json]: Miniguns are rifles and not launchers and they're less similar to rifles than big xbows to small ones.

Comment (kevingranade) [inline data/json/items/archery.json]: The aspects I'm thinking of are bulk, weight and inaccuracy, Gantz has all of thse in spades, it's essentially a siege engine you can barely carry.

Basically if no one has an argument for switching it to launcher I'll leave it using the rifle skill :)

Honestly the minigun probably shouldn't use rifle either, but I don't think launcher is any better, nothing really fits at all.

**Comment by DangerNoodle** [comment]
You took the point I presented in that pull request, and went in the absolute opposite direction.

**Comment by Night-Pryanik** [comment]
I agree that crossbows are operated in a rifles manner, not bows. Both should be raised up to head, stock should rest at the shoulder. Bows are aimed and fired in a different manner.

**Comment by DangerNoodle** [comment]
Very well then. It at least will be closer to consistent than current handling.

Reminder that Vehicle Additions Pack has the scopio ballista, and that More Survival Tools has the makeshift crossbow.

**Comment by DangerNoodle** [comment]
In addition, remember that professions that start with crossbows will like have archery skill, this will have to be changed.

**Pull request merged**